interface/modules/custom_modules/oe-module-claimrev-connect/public/index.php

```diff
@@ -0,0 +1,57 @@
+<?php
+    $tab="home";
+?>
+<html>
+    <head>
+        <link rel="stylesheet" href="../../../../../public/assets/bootstrap/dist/css/bootstrap.min.css">
+    </head>
+<title>ClaimRev Connect - Home</title>
+<body>
+    <div class="row"> 
+        <div class="col">
+            <?php
+                include '../templates/navbar.php';
+            ?>
+        </div>
+    </div>
+    <div class="row"> 
+        <div class="col">
+            <h1>ClaimRev - Client Access</h1>        
+        </div>
+    </div>
+    <div class="row"> 
+        <div class="col">
+            <div class="card">
+                <p>
+                    Welcome to the ClaimRev Connector. This is your link to the portal and claim processing.
+
+                </p>
+                <h6>Tab Descriptions</h6>
+                <ul>
+                    <li>
+                        Claims -> Lets you search claims sent to ClaimRev and view the status.
+                    </li>
+                    <li>
+                        X12 Tracker -> Display's files that have been submitted or in the process of being submitted.
+                    </li>
+                    <li>
+                        Setup -> Helps identify any setup issues along with checking background services.
+                    </li>
+                    <li>
+                        Connectivity -> Display's information that may help support fix connection issues you maybe having.
+                    </li>
+                </ul>
+
+                <h6>Support/Sales</h6>
+                <ul>
+                    <li> Call: <a href="tel:9189430020">1-918-943-0020<a>   </li>
+                    <li> Email Support: <a href = "support@claimrev.com">support@claimrev.com</a> </li>
+                    <li> Email Sales: <a href = "sales@claimrev.com">sales@claimrev.com</a> </li>
```

Comment:
xlt() string above. here are several examples of above lines:
```
<?php echo xlt("Welcome to the ClaimRev Connector. This is your link to the portal and claim processing."); ?>
<li> <?php echo xlt("Email Support"); ?>: <a href = "support@claimrev.com">support@claimrev.com</a> </li>
```

Reply:
done